### PR TITLE
Fix JSON serialization BigInt exception when parsing call service response

### PR DESCRIPTION
**Pull request by smbera**
**User-Facing Changes**

**Description**
image

**Example**
JSON.parse BigInt Wrong:
```
JSON.parse('{ "value" : 9223372036854775807 }')
// {value: 9223372036854776000}
```
JSON.stringify BigInt Wrong:
```
JSON.stringify({ value: 1n })
// Uncaught TypeError: Do not know how to serialize a BigInt
```

**Links**
[What is the equivalent of bigint in C](https://stackoverflow.com/questions/2113526/what-is-the-equivalent-of-bigint-in-c)
[Use within JSON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#use_within_json)

### CODE REVIEW

File: packages/studio-base/src/panels/CallService/CallService.tsx [hunk 1]
```diff
@@ -68,7 +70,7 @@ function parseInput(value: string): { error?: string; parsedObject?: unknown } {
   let parsedObject;
   let error = undefined;
   try {
-    const parsedAny: unknown = JSON.parse(value);
+    const parsedAny: unknown = JSONbigNative.parse(value);
```
[the remark this answers is not in the document]
Comment (smbera) [reply packages/studio-base/src/panels/CallService/CallService.tsx]: done

File: packages/studio-base/src/panels/CallService/CallService.tsx [hunk 2]
```diff
@@ -178,9 +180,9 @@ function CallServiceContent(
       setState({ status: "requesting", value: `Calling ${config.serviceName}...` });
       const response = await context.callService(
         config.serviceName!,
-        JSON.parse(config.requestPayload!),
+        JSONbigNative.parse(config.requestPayload!),
```
[the remark this answers is not in the document]
Comment (smbera) [reply packages/studio-base/src/panels/CallService/CallService.tsx]: Done

File: packages/studio-base/src/panels/CallService/CallService.tsx [hunk 3]
```diff
@@ -178,9 +180,9 @@ function CallServiceContent(
...
       );
-      setState({ status: "success", value: JSON.stringify(response, undefined, 2) ?? "" });
+      setState({ status: "success", value: JSONbigNative.stringify(response, undefined, 2) ?? "" });
```
[the remark this answers is not in the document]
Comment (smbera) [reply packages/studio-base/src/panels/CallService/CallService.tsx]: Done

**Comment by smbera** [comment]
> From your screenshot, it looks like you are using a custom call-service extension? Also what data source were you connected to?
> 
> About the PR, I have requested a few changes, mainly dropping the new dependency and using a replacer function to handle BigInt in `JSON.stringify`.

Done

**Review by achim-k — APPROVED** [review]

**Review by achim-k — APPROVED** [review]

**Pull request merged**